src/main/resources/ca/corefacility/bioinformatics/irida/model/workflow/analysis/type/workflows/SNVPhyl/1.2.2/irida_workflow.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<iridaWorkflow>
+	<id>f609c177-c268-4ad0-9d7f-9f9d5187fef7</id>
+	<name>SNVPhyl</name>
+	<version>1.2.2</version>
+	<analysisType>PHYLOGENOMICS</analysisType>
+	<inputs>
+		<sequenceReadsPaired>sequence_reads_paired</sequenceReadsPaired>
+		<reference>reference</reference>
+		<requiresSingleSample>false</requiresSingleSample>
+	</inputs>
+	<parameters>
+		<parameter name="enable-density-filter" required="true">
```

Comment:
I would strongly prefer to keep this as a required option. My reason is that I suspect some people will run SNVPhyl with the defaults and just use the results as-is, without checking to make sure if the results make sense. By default (previously) if SNV density filtering is turned on this could lead to the majority of the SNVs being removed from their results and lead to invalid results. I would rather force people to make this selection themselves than hide it as a default option.

There could maybe be a help icon we add which lets people hover over and read more information about this option. But that would be a UI change I'd have to look into making.

Reply:
Fixed in 2546587f6f3600d52e8e398d1f28bf81ccd8cbf8. I also added a link to the docs in the SNVPhyl description on IRIDA.

Reply:
Great, thanks.  That's a good interim solution to linking to those docs as well.